extensions/ql-vscode/src/packaging.ts

```diff
@@ -123,8 +122,8 @@ export async function handleInstallPacks(
     }
     if (failedPacks.length > 0) {
       void logger.log(`Errors:\n${errors.join('\n')}`);
-      void showAndLogWarningMessage(
-        `Unable to install some packs: ${failedPacks.join(', ')}. See logs for more details.`
+      throw new Error(
+        `Unable to install packs: ${failedPacks.join(', ')}. See logs for more details.`
```

Comment:
nit: Maybe make it singular since the button below says "Show log".

```suggestion
        `Unable to install packs: ${failedPacks.join(', ')}. See log for more details.`
```